keep/workflowmanager/workflowscheduler.py

```diff
@@ -413,6 +413,7 @@ def _timeout_workflows(self):
         """
         Record timeout for workflows that are running for too long.
         """
+        is_running_in_keep_cloud = "keephq.dev" in config("KEEP_API_URL")
```

Comment:
There is an env for that already in consts

https://github.com/keephq/keep/blob/6816f6b533d9ead111dd84b6a1db14ed1f8919f8/keep/api/consts.py#L8